app/ui-login/client/reset-password/resetPassword.js

```diff
@@ -3,13 +3,16 @@ import { Accounts } from 'meteor/accounts-base';
 import { FlowRouter } from 'meteor/kadira:flow-router';
 import { Template } from 'meteor/templating';
 import toastr from 'toastr';
+import { ReactiveDict } from 'meteor/reactive-dict';
 
-import { modal } from '../../../ui-utils';
+import { modal, call } from '../../../ui-utils/client';
 import { t } from '../../../utils';
-import { Button } from '../../../ui';
 import { callbacks } from '../../../callbacks';
 
 Template.resetPassword.helpers({
+	disabled() {
+		// return Template.instance().state.get('password') ? '' : 'disabled';
```

Comment:
Why?

Reply:
to force the backend to receive an empty password and accept/save